### DISCUSSION TITLE
2 Way Sync

### DISCUSSION THREAD

**Discussion by Drakenberg** [Vaccarini-Lorenzo/iCalObsidianSync#25] [General]
Does this sync from ical to obsidian as well as from obsidian to ical?

**Comment by Vaccarini-Lorenzo** [comment]
Hi there! Using the inline event view syntax you can embed your calendar events in your note simply writing a code block as such  ```<ical> from:YYYY-MM-DD to:YYYY-MM-DD```
If you use the release 1.1.5 the inline event view is not synced in real time and you'll need to render once again the page.
Otherwise you could clone the inline_event_view_development branch and you'd get the real-time sync and also the possibility to move the events around.

EDIT: The dev version is still a beta and has some issues like disconnections and possible plugin crashes, hopefully I'll work on it in the next future.

**Comment by Vaccarini-Lorenzo** [comment]
Hello @Drakenberg, the two way sync is now available thanks to the inline event view feature for Apple Calendar. Check the readme for more informations
